bin/ext/cli.sh

```diff
@@ -28,7 +28,7 @@ updateCli() {
     CLASS=org.apache.hadoop.hive.cli.CliDriver
     JAR=hive-cli-*.jar
   else
-    export HADOOP_CLIENT_OPTS=" -Dproc_beeline $HADOOP_CLIENT_OPTS -Dlog4j.configurationFile=beeline-log4j2.properties"
+    export HADOOP_CLIENT_OPTS="-Dproc_beeline -Dorg.slf4j.simpleLogger.defaultLogLevel=warn $HADOOP_CLIENT_OPTS"
```

Comment:
dumb question? how would users enable debug logging in beeline with this approach ?

Reply:
Not a dumb question.  I think I included it in the original JIRA, but here it is:

```
export HADOOP_CLIENT_OPTS="-Dorg.slf4j.simpleLogger.defaultLogLevel=debug"
beeline ...
```